app/file-upload/server/config/FileSystem.js

```diff
@@ -6,6 +6,9 @@ import _ from 'underscore';
 
 import { settings } from '../../../settings';
 import { FileUploadClass, FileUpload } from '../lib/FileUpload';
+import { getFileRange, setRangeHeaders } from '../lib/ranges';
+
+const statSync = Meteor.wrapAsync(fs.stat);
```

Comment:
Isn't a `fs.statSync` function? 👀

Reply:
yes. but using `Meteor.wrapAsync` it doesn't make it really sync, it is still doesn't block the event loop, which is better 😬  take a look at https://blog.meteor.com/meteor-fibers-meet-promises-meet-callbacks-a-practical-guide-e134db15afcb

Reply:
Ahhhh! Nice nice, I really thought you wanted to do it "sync" by the name haha.